downstream/modules/platform/con-ocp-supported-install.adoc

```diff
@@ -7,9 +7,30 @@ You can use the OperatorHub on the {OCP} web console to install {OperatorPlatfor
 
 Alternatively, you can install {OperatorPlatform} from the {OCPShort} command-line interface (CLI), `oc`.
 
-Follow one of the workflows below to install the {OperatorPlatform} and use it to install the components of {PlatformNameShort} that you require.
+After you have installed {OperatorPlatform} you must create an *AnsibleAutomationPlatform* custom resource (CR). This allows you to manage {PlatformNameShort} components from a single unified interface known as the platform gateway. As of version 2.5, you must create an {PlatformNameShort} CR, even if you have an existing automation controller,  automation hub, or Event-driven Ansible (EDA), components.
 
-* {ControllerNameStart} custom resources first, then {HubName} custom resources;
-* {HubNameStart} custom resources first, then {ControllerName} custom resources;
-* {ControllerNameStart} custom resources;
-* {HubNameStart} custom resources.
+If existing components have already been deployed, you must specify these components on the {PlatformNameShort} CR. You must create the custom resource in the same namespace as the existing components.
+
+[cols=2*a,options="header"]
+|===
+| *Supported scenarios* | *Supported scenarios with existing components*
+|
+* {PlatformNameShort} CR for blank slate install with automation controller, automation hub, and EDA enabled
```

Comment:
Just a suggestion but you could use attributes for automation controller, hub and EDA 👍

Reply:
Done!